.rustfmt.toml

```diff
@@ -0,0 +1,24 @@
+# stable options
+use_small_heuristics = "Default"
+force_explicit_abi = true
+hard_tabs = false
+max_width = 100
+merge_derives = false
+newline_style = "Auto"
+remove_nested_parens = true
+reorder_imports = true
+reorder_modules = true
+tab_spaces = 4
+use_field_init_shorthand = true
+use_try_shorthand = true
+edition = "2018"
+
+# unstable options (for one-off fixes and CI)
+merge_imports = true
+wrap_comments = true
+comment_width = 80
+condense_wildcard_suffixes = true
+format_strings = true
+normalize_comments = true
```

Comment:
The default is `false` but I changed it to `true` because I like the consistency of having all comments look the same.  I'm not sure of other editors, but in my editor, if I select a block of code and use the keystroke to comment it, it inserts `//` at the beginning of every line, instead of using the `/* */` variant.  My hunch is that other editors work similarly, though I'll admit total ignorance on this point.  I also don't feel strongly about this one way or the other.
https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#normalize_comments